source/Irrlicht/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ endif()
 
 option(FAST_MATH "Enable fast math" ON)
 
+find_package(OpenSSL REQUIRED)
```

Comment:
Yes, but note that `OPENSSL_LIBRARIES` is not a valid configuration option; use `OPENSSL_CRYPTO_LIBRARY` (`-DOPENSSL_CRYPTO_LIBRARY=/path/to/crypto.so`) and `OPENSSL_SSL_LIBRARY` instead. `cmake-gui` shows the available configuration options (from terminal, use `cmake -LAH <path-to-build-directory>` instead)

Reply:
what about where it will get include files for openssl from?

Reply:
@devshgraphicsprogramming from `OPENSSL_INCLUDE_DIR`. On my system, that’s just `/usr/include` by default, so that you can write e.g. `openssl/aes.h` to reference `/usr/include/openssl/aes.h`. That’s the way libraries are used with CMake.